sql/catalyst/src/main/java/org/apache/spark/sql/connector/catalog/SupportsAtomicPartitionManagement.java

```diff
@@ -82,4 +82,27 @@ void createPartitions(
    * @return true if partitions were deleted, false if any partition not exists
    */
   boolean dropPartitions(InternalRow[] idents);
+
+  /**
+   * Drop an array of partitions atomically from table.
+   * <p>
+   * If any partition doesn't exists,
```

Comment:
```suggestion
   * If any partition doesn't exist,
```

Reply:
Thank you. This is a copy-paste from https://github.com/apache/spark/blob/60fa8e304d1284c269b8ac6be9a3fd65197ef6ec/sql/catalyst/src/main/java/org/apache/spark/sql/connector/catalog/SupportsAtomicPartitionManagement.java#L78 . Let me fix this there too.